pkg/scheduler/framework/plugins/noderesources/fit.go

```diff
@@ -106,8 +104,18 @@ func (f *Fit) Filter(ctx context.Context, cycleState *framework.CycleState, pod
 	if err != nil {
 		return framework.NewStatus(framework.Error, err.Error())
 	}
-	_, reasons, err := predicates.PodFitsResourcesPredicate(pod, r, f.ignoredResources, nodeInfo)
-	return migration.PredicateResultToFrameworkStatus(reasons, err)
+	reasons := predicates.PodFitsResourcesPredicate(pod, r, f.ignoredResources, nodeInfo)
```

Comment:
Would we remove `predicates.PodFitsResourcesPredicate` deps in NodeResourcesFit filter plugin